app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :recoverable, :validatable
```

Comment:
Can you provide a quick summary of what these traits will allow on the user? And you can remove the comment!

Reply:
From the docs:

Database Authenticatable: hashes and stores a password in the database to validate the authenticity of a user while signing in. The authentication can be done both through POST requests or HTTP Basic Authentication.
Registerable: handles signing up users through a registration process, also allowing them to edit and destroy their account.
Recoverable: resets the user password and sends reset instructions.
Validatable: provides validations of email and password. It's optional and can be customized, so you're able to define your own validations.

Reply:
Looks like we can get rid of Recoverable since we decided earlier that we don't want to be able to reset account details.